src/i18n/en.js

```diff
@@ -135,18 +135,24 @@ const messages_en = {
     new: `New`,
     recaptcha_legal_MD: `
 Site protected by reCAPTCHA and the Google [Privacy Policy](https://policies.google.com/privacy?hl=en) and [Terms of Service](https://policies.google.com/terms?hl=en).`,
+    save: `Save`,
     send: `Send`,
     show: `Show`,
     synchronizing: `Synchronizing`,
     ui_library_image: `https://cdn.fromdoppler.com/doppler-ui-library/${dopplerUiLibraryVersion}/img/{imageUrl}`,
     unexpected_error: `Ouch! An unexpected error occurred, please try again`,
   },
   contact_policy: {
+    amount_description: 'Do not send more than',
+    amount_unit: 'Emails',
```

Comment:
'Emails' is a general text. 
It could be good to have a general section in the translations file, where the texts that are common on the platform are placed. In this way, it is avoided that the translation files grow very large when having many repeated texts. @andresmoschini

Reply:
In general, I agree with you.

But, in this case, `Emails` does not represent only "emails", it represents the _amount unit_, I mean, it could be "emails", "envíos", "deliveries", "contacts", "events". So, in my opinion, it should not be in a general section.

Reply:
I agree, this text can change.

Reply:
In fact we have a (common) section where common texts are placed. We can add 'Emails' as a general text.

Reply:
Yes, we could. But in my opinion, it is not a `General.Email`, it is a `contact_policy.amount_unit`